src/proxy/mysqlProxy.lua

```diff
@@ -197,7 +197,6 @@ function queryType()
     -- Detects if query can be handled locally without sending it to qserv
     local isLocal = function(qU)
         if (string.find(qU, "^SHOW ") and not string.find(qU, "^SHOW .*PROCESSLIST")) or
-           string.find(qU, "^SET ") or
```

Comment:
So, this forwards all "SET" queries to qserv instead of mysql? I'm a bit worried about that, there _may be_ some mysql options that people may want to change with the SET queries, and stuff like JDBC could also issue SET queries to change some session-level behavior. Maybe it's better to still forward all SET queries to mysql but send qserv-specific options to qserv. Probably similarly to the line above change this one to:

    (string.find(qU, "^SET ") and not string.find(qU, "^SET .*QSERV")) or